lib/services/local/presence.go

```diff
@@ -1727,6 +1730,16 @@ func backendItemToDatabaseServer(item backend.Item) (types.ResourceWithLabels, e
 	)
 }
 
+// backendItemToDatabaseService unmarshals `backend.Item` into a
+// `types.DatabaseService`, returning it as a `types.ResourceWithLabels`.
+func backendItemToDatabaseService(item backend.Item) (types.ResourceWithLabels, error) {
+	return services.UnmarshalDatabaseService(
+		item.Value,
+		services.WithResourceID(item.ID),
+		services.WithExpires(item.Expires),
+	)
+}
```

Comment:
I see other keep alive types here in `presence.go`. should the database service be here as well?

Reply:
Yes :+1:
We also need to change its return values to include the `types.KeepAlive`